README.md

```diff
@@ -5,7 +5,7 @@
 ## Overview
 
 kintone output plugin for Embulk stores app records from kintone.
-embulk 0.9 is only supported due to the dependency of kintone-java-sdk 0.4.0, which requires java 8
+embulk 0.9 is only supported due to the dependency of [kintone-java-client](https://github.com/kintone/kintone-java-client), which requires java 8.
```

Comment:
You can delete this comment. The motivation of this comment was that why didn't we choose the latest kintone-java-sdk. kintone-java-client supports Java from 8.

Reply:
I see.
I deleted it.